organizations/tests.py

```diff
@@ -366,36 +366,36 @@ def test_organizations_users_get(self):
     def test_organizations_courses_get(self):
         organization = self.setup_test_organization()
         courses = CourseFactory.create_batch(2)
-        groups = GroupFactory.create_batch(2)
+        users = UserFactory.create_batch(2)
 
         for i in xrange(2):
```

Comment:
I think it makes more sense here to iterate over users list in for loop like this `for user in users:`